tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_cpu

```diff
@@ -1,5 +1,6 @@
 FROM quay.io/pypa/manylinux2014_x86_64:latest
 
+ARG USE_JDK11=0
```

Comment:
Did you pass this argument down to install_centos.sh?

Reply:
ARG is also an environment variable